src/components/DirectAnswer.tsx

```diff
@@ -59,8 +59,8 @@ const builtInCssClasses: Readonly<DirectAnswerCssClasses> = {
  * @returns A react element for DirectAnswer
```

Comment:
nit: can you update the doc here to refer to "Search API"? and similarly for the `SpellCheck` component's docs